mutornadomon/monitor.py

```diff
@@ -70,6 +107,12 @@ def __init__(self, host_limit=r'.*', request_filter=LOCALHOST, io_loop=None):
         self._GAUGES = {}
         self._reset_ephemeral()
 
+    def _publish(self):
+        try:
+            self.publisher(self.metrics)
```

Comment:
I was thinking the 2 instrumentation methods will be mutually exclusive. You either call `instrument_ioloop` or `initialize_mutornadomon`. I will update the README as well.